### ISSUE TITLE
Speed up/down portion of CW message

### ISSUE THREAD

**Issue by vazquezrodjoel** [foldynl/QLog#491]
It would be great if we could setup a speed up or down "flag" on the CW messages memory/macros.  

For example say I have my key set to 26 WPM and the following macro:
`CQ CQ CQ DE <MYCALL> <MYCALL> <dwn20><MYCALL> </dwn20> K
`
so to make the software slow down the speed when sending the callsign last time. 

Or if I if slow:
`<DXCALL> <RSTN> <up24> <RSTN> </up24> K`
so it will solow down for that specific report.

Could be with a specific speed (like <26wpm> or specifying a speed change (up5wpm).

CQrlog does it by + or - sign in the macro:
```
+ or - in macro text will increase/decrease CW speed by 5WPM/one mark.
"TU +++599--- 001" will send 599 with 15WPM higer speed. 
```

**Comment by foldynl** [comment]
I don't have much experience with CW under CQRLog, but according to the source code, it seems that this feature only works for K3NG, CWDaemon, and Winkey. It’s not supported for hamlib, is that correct? Honestly, I can't imagine how I would implement this for the user, where some macros would only work for selected CW interfaces. The only thing that comes to mind is to ignore the new macros in the case of MorseOverCAT (Hamlib CW Interface).

**Comment by vazquezrodjoel** [comment]
Well, Im not sure, but at least I use it with CWDaemon and it works great on it.  Now that you mention it I dont recall ever using it on hamlib directly... it might work, just dont have experience with that.  I can play with that later this week as I still have the cqr installed on the computer.  Ill play with it and let you know.  

I have never had a K3NG or Winkey, son cant talk about those, but your suggestion sounds reasonable for me of ignoring those for morseovercat if that is selected. Im sorry, I work in IT and very familiar with linux myself, but while I understand basics concept of it, I'm not a programmer perse :).

**Comment by vazquezrodjoel** [comment]
Actually, I have been playing with cqr tonight for a while trying to make that work, but it doesnt even sent cw over hamlib... at all... :|

**Comment by foldynl** [comment]
Thank you for confirming. According to the source code, it appears so. I’ll need to consider how this could be implemented, as it cannot be done with Hamlib Morse. The options are:
1) Ignore it, though this might be confusing for users.
2) Notify the user that it’s not possible with Hamlib.

**Comment by vazquezrodjoel** [comment]
Yeah, maybe a message if they select cw over cat with hamlib that that shortcut/option will not work.

I was also thinking that, depending how its implemented, this could server like a quick setup of specific speed.  For example I usually leave it by default as 20wpm, and I don't use all the memory F# keys.  So I could use one to speedup the speed to say 26wpm, so I don't have to click on the speed buttons.  If that make sense...
